stac_fastapi/types/setup.py

```diff
@@ -6,7 +6,7 @@
     desc = f.read()
 
 install_requires = [
-    "fastapi-slim>=0.111.0",
+    "fastapi>=0.100.0",
```

Comment:
```suggestion
    "fastapi>=0.109.0",
```

because we use new features introduced in starlette 0.29